hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
In all of our scripts it's customary to use this in the preamble:

```sh
set -o errexit
set -o nounset
set -o pipefail
```

The first option exits the script on an error, the second errors when a variable is used before it's assigned, the third fails the entire pipe chain if any member fails.